### ISSUE TITLE
Don't Show the Image Caption If the Caption Is Empty

### ISSUE THREAD

**Issue by href** [OneGov/onegov.town#114]
The image caption is visible, even if the alt text is empty (`alt=""`). This should be avoided because it results in an empty grey box.

**Comment by freinhard** [comment]
![bildschirmfoto 2015-11-25 um 10 17 46](https://cloud.githubusercontent.com/assets/186856/11393044/cb3cf5a2-935d-11e5-8cfe-2ba4cfef9753.png)
